tests/bugs/glusterd/daemon-log-level-option.t

```diff
@@ -56,10 +57,12 @@ EXPECT 0 Trace_messages_count "/var/log/glusterfs/glustershd.log"
 # stop the volume and remove glustershd log
 TEST $CLI volume stop $V0
 rm -f /var/log/glusterfs/glustershd.log
+EXPECT_WITHIN ${PROCESS_UP_TIMEOUT} "0" online_brick_count
```

Comment:
Maybe we should create a new timeout value for stopping volumes ? stop should be much faster than start, so probably we should use a smaller timeout.